lib/src/rules/prefer_equal_for_default_values.dart

```diff
@@ -7,7 +7,7 @@ import 'package:analyzer/dart/ast/token.dart';
 import 'package:analyzer/dart/ast/visitor.dart';
 import 'package:linter/src/analyzer.dart';
 
-const _desc = r'Prefer equal for default values.';
+const _desc = r'Use equal instead of colon when specifying default values.';
 
```

Comment:
As you suggest, perhaps a link to the style guide in the details?

See, for example:

https://github.com/dart-lang/linter/blob/36bc8fbf15e0156a8f25d87d60cffd41b412f892/lib/src/rules/camel_case_types.dart#L14